docs/code-scanning-tools/sonar.md

```diff
@@ -1,22 +1,26 @@
 ---
-title: "Sonar"
+title: "SonarCloud"
 sidebar_position: 1
 ---
 
-# Sonar
+# SonarCloud
 
-Pixeebot can automatically fix issues detected by [Sonar](https://www.sonarsource.com/products/sonarcloud/). This page explains how to integrate your Sonar results with Pixeebot.
+Pixeebot can automatically fix issues detected by [SonarCloud](https://www.sonarsource.com/products/sonarcloud/). This page explains how to integrate your SonarCloud results with Pixeebot.
+
+:::info
+Looking to fix [SonarQube](https://www.sonarsource.com/products/sonarqube/) issues? See [SonarQube](./sonarqube.md)
+:::
```

Comment:
Put this here to help SonarQube users that land here because they were given the link https://pixee.ai/sonar